dt/PIDONE/DFG-pidone-18.0-3cont-2comp-ipv4-geneve-HA

```diff
@@ -0,0 +1,43 @@
+RHOS Deployed Topology DFG-pidone-18.0-3cont-2comp-ipv4-geneve-HA
+Based on OpenStack K8S operators from the "main" branch of the OpenStack Operator repo on Oct 17th, 2023
+
+General information:
+Revision	Change		Publication Date
+v0.1	 	Initial		Dec 20 2023
+
+Node topology:
+Node role	                                bm/vm	amount
+Openshift master/worker combo-node cluster	vm	    3
+Compute node					vm	    3
+Galera DB node, replicating	                vm	    3
```

Comment:
Can this exist in the OCP control plane or are these 3 additional VM's needed for the deployment?

Reply:
The Galera cluster can be deployed on the OCP Control Plane. I'll send a change soon.